shared/src/question-generators/math/modularArithmetic/eea.ts

```diff
@@ -0,0 +1,234 @@
+import { Language } from "@shared/api/Language"
+import {
+  MultiFreeTextFeedbackFunction,
+  MultiFreeTextFormatFunction,
+  MultiFreeTextQuestion,
+  QuestionGenerator,
+} from "@shared/api/QuestionGenerator"
+import { serializeGeneratorCall } from "@shared/api/QuestionRouter"
+import Random from "@shared/utils/random"
+import { t, tFunctional, Translations } from "@shared/utils/translations"
+
+const translations: Translations = {
+  en: {
+    name: "Extended Euclidean Algorithm",
+    description:
+      "Fill in the steps to calculate the GCD of two numbers using the Extended Euclidean Algorithm.",
+    eeaQuestion: "Calculate the GCD of ${{a}}$ and ${{b}}$ by completing the following equations:",
+    linearCombinationPrompt: "Express the GCD as a linear combination:",
+    gcd: "gcd",
+    feedbackInvalid: "Input must be an integer.",
+  },
+  de: {
+    name: "Erweiterter Euklidischer Algorithmus",
+    description:
+      "Fülle die Schritte aus, um den GGT zweier Zahlen mit dem erweiterten euklidischen Algorithmus zu berechnen.",
+    eeaQuestion:
+      "Berechne den GGT von ${{a}}$ und ${{b}}$ durch das Vervollständigen der folgenden Gleichungen:",
+    linearCombinationPrompt: "Drücke den GGT als Linearkombination aus:",
+    gcd: "ggt",
+    feedbackInvalid: "Eingabe muss eine ganze Zahl sein.",
```

Comment:
It feels like this should start with an article to have a full sentence.
maybe this also applies to the English version.